capture-sdk/sdk/src/main/java/net/gini/android/capture/camera/CameraFragmentListener.java

```diff
@@ -21,7 +21,7 @@
  *
  * @suppress
  */
-interface CameraFragmentListener {
+public interface CameraFragmentListener {
```

Comment:
[Request] Please don't alter classes in the `sdk` module yet. You can unintentionally alter the SDK's public api. For example this interface should not be made public.

Reply:
Please ignore the above. It needs to be made public because the `DocumentCheckResultCallback` needs to be accessible from Kotlin. Strange that it was accessible from Java even though `CameraFragmentListener` was package private.